.evergreen/config.yml

```diff
@@ -1,2116 +1,1945 @@
+    tasks:
+      - .in-use-encryption
+
+  - name: in-use-encryption-openssl
+    display_name: "In-Use Encryption (OpenSSL)"
+    # patchable: false
+    run_on:
+      - rhel80-small
+    expansions:
+      LIBMONGOCRYPT_OS: rhel-80-64-bit
+      AUTH: auth
+      SSL: ssl
+      OPENSSL: true
+    tasks:
+      - test-in-use-encryption-openssl
+
+  - name: in-use-encryption-serverless
+    # TODO: Enable this build variant when the serverless issues are fixed.
+    disable: true
+    # patchable: false
+    display_name: In-Use Encryption (Serverless)
+    run_on:
+      - rhel87-small
+    tasks:
+      - test-in-use-encryption-serverless
+
+  - name: load-balancer
+    display_name: "Load Balancer"
+    run_on:
+      - rhel87-small
+    tasks:
+      - .load-balancer
+
+###############
+# Task Groups #
+###############
+task_groups:
+  - name: serverless-task-group
+    setup_group_can_fail_task: true
+    setup_group_timeout_secs: 1800 # 30 minutes
+    setup_group:
+      - func: "fetch source"
+      - func: "create expansions"
+      - func: "prepare resources"
+      - func: "windows fix"
+      - func: "fix absolute paths"
+      - func: "init test-results"
+      - func: "make files executable"
+      - func: "install rust"
+      - command: shell.exec
+        params:
+          shell: "bash"
+          script: |
+            ${PREPARE_SHELL}
+            set +o xtrace
+            SERVERLESS_DRIVERS_GROUP=${SERVERLESS_DRIVERS_GROUP} \
+            SERVERLESS_API_PUBLIC_KEY=${SERVERLESS_API_PUBLIC_KEY} \
+            SERVERLESS_API_PRIVATE_KEY=${SERVERLESS_API_PRIVATE_KEY} \
+            LOADBALANCED=ON \
+              bash ${DRIVERS_TOOLS}/.evergreen/serverless/create-instance.sh
+      - command: expansions.update
+        params:
+          file: serverless-expansion.yml
+    teardown_group:
+      - command: shell.exec
+        params:
+          script: |
+            ${PREPARE_SHELL}
+            set +o xtrace
+            SERVERLESS_DRIVERS_GROUP=${SERVERLESS_DRIVERS_GROUP} \
+            SERVERLESS_API_PUBLIC_KEY=${SERVERLESS_API_PUBLIC_KEY} \
+            SERVERLESS_API_PRIVATE_KEY=${SERVERLESS_API_PRIVATE_KEY} \
+            SERVERLESS_INSTANCE_NAME=${SERVERLESS_INSTANCE_NAME} \
+              bash ${DRIVERS_TOOLS}/.evergreen/serverless/delete-instance.sh
+      - func: "upload test results"
+      - func: "upload-mo-artifacts"
+      - func: "cleanup"
+
+    tasks:
+      - "test-serverless"
+
+  - name: azure-kms-task-group
+    setup_group_can_fail_task: true
+    setup_group_timeout_secs: 1800 # 30 minutes
+    setup_group:
+      - func: "fetch source"
+      - func: "create expansions"
+      - func: "prepare resources"
+      - func: "windows fix"
+      - func: "fix absolute paths"
+      - func: "init test-results"
+      - func: "make files executable"
+      - func: "install rust"
+      - func: "install libmongocrypt"
+      - command: shell.exec
+        params:
+          shell: bash
+          script: |-
```

Comment:
For my own education, what's the difference between `script: |` and `script: |-` ?

Reply:
> For my own education, what's the difference between `script: |` and `script: |-` ?

From https://yaml.org/spec/1.2.2/#8112-block-chomping-indicator: 

> Stripping is specified by the “-” chomping indicator. In this case, the final [line break](https://yaml.org/spec/1.2.2/#line-break-characters) and any trailing [empty lines](https://yaml.org/spec/1.2.2/#empty-lines) are excluded from the [scalar’s content](https://yaml.org/spec/1.2.2/#scalar).

https://yaml-multiline.info has a neat interactive explainer.